Make it installable codelab: Warn about installability criteria (#9773)
GoogleChrome/web.dev: src/site/content/en/progressive-web-apps/codelab-make-installable/index.md

});
```

{% Aside 'gotcha' %}

> The `beforeinstallevent` event won't fire, and the "Install" button won't
> appear, until you click in the page and wait 30 seconds, in order to meet
> [the PWA installability criteria](/install-criteria/).
> 
> You'll also need to open the app in its own browser tab, and not in an
> iframe. (The lab itself will warn you if it's running in an iframe.)
> {% endAside %}

Reviewer: ```suggestion
You must click into the page and wait 30 seconds before the `beforeinstallevent` fires, and the **Install** button appears. This is to meet [the PWA installability criteria](/install-criteria/).

In addition, you need to open the app in its own browser tab, and not in an iframe. The lab itself will warn you if it's running in an iframe.
{% endAside %}
```
